ansible/inventory/testing/group_vars/all/vars.yml

```diff
@@ -7,3 +7,21 @@ website_host: testing.privacylx.org
 # Lets Encrypt
 le_email: admin@privacylx.org
 le_staging: true
+
+# Login, SSH public key, comment and state of users with admins rights
+# State absent removes the user account
+adm_acct:
+  andz:
+    login: andz
+    comment: anadahz
+    state: present
+    sshkey: 'ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIOVYO2Xu1pKrax98FpZ3dauRDMqE8Z4IPSYihD5XGRaO'
+  testuser1:
+    login: testuser1
+    comment: testuser1
+    state: absent
```

Comment:
is this intended to be absent? If so, I would suggest you remove this test user manually and removing it from here. I think it doesn't add much to the script if it's only every going to be present once.

Reply:
This is in fact intended to be absent.
What if we forget to remove a previously added user manually?
Are we ever going to find out that this user hasn't been removed (since this is not recorded in a git log or anything)?

Ideally Ansible should be instructed to only allow specific accounts but this is AFAIK currently not possible. Feel free to suggest a better way to do this in an automated, idempotent and error prone way.

Reply:
You're right. Then I propose we do all of the user stuff in `users.yml` and then we include that in the `vars.yml`. Is that possible?

Reply:
Any specific reasoning behind this?
The file has currently very few lines and it's very easy to read and review.

Reply:
Yes, each login statement is 22 lines long. Adding mine, then we have 44 already and another one will make it bigger. If we separate this, it will be that much cleaner

Reply:
Resolved in https://github.com/PrivacyLx/devops/pull/21/commits/12503c7f1b9496d5b3a40369ffc8182713b0245e